keyboards/yetis/info.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "jogme",
+    "keyboard_name": "Yetis",
+    "maintainer": "qmk",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "direct": [
+            [null, null, "F7", "F6", "F5", "F4"],
+            ["F0", "F1", "B1", "B3", "B2", "B6"],
+            [null, "C7", "D5", "B7", "B5", "E6"],
+            [null, null, null, null, "D7", "C6"]
+        ]
+    },
+    "processor": "RP2040",
```

Comment:
```suggestion
```

Reply:
If I remove this line or the "bootloader", qmk does not compile as:

```
* Invalid bootloader specified. Please set an appropriate bootloader in your rules.mk or info.json.   [ERRORS]
```